apps/site/components/ApiDocs/StabilityIndex/index.module.css

```diff
@@ -0,0 +1,48 @@
+.stabilityIndex {
+  @apply flex
+    flex-row
+    items-center
+    gap-2
+    rounded
+    px-4
+    py-3
+    text-white;
```

Comment:
Is the font-size supposed to be 16px? I thought it was 14px default?

Reply:
I'll check that now